test/integration/render_task_test.rb

```diff
@@ -0,0 +1,191 @@
+# frozen_string_literal: true
+require 'test_helper'
+require 'kubernetes-deploy/render_task'
+
+class RenderTaskTest < KubernetesDeploy::TestCase
+  include FixtureDeployHelper
+
+  def test_render_task
+    render = build_render_task(fixture_path('hello-cloud'))
+    fixture = 'configmap-data.yml'
+
+    assert_render_success(render.run([fixture], test_output_stream))
+
+    stdout_assertion do |output|
+      assert_equal output, <<~RENDERED
+        ---
+        apiVersion: v1
+        kind: ConfigMap
+        metadata:
+          name: hello-cloud-configmap-data
+          labels:
+            name: hello-cloud-configmap-data
+            app: hello-cloud
+        data:
+          datapoint1: value1
+          datapoint2: value2
+      RENDERED
+    end
+
+    assert_render_success(render.run([fixture, fixture], test_output_stream))
```

Comment:
This is probably rendering the same thing twice, right? Should we be de-duping the file names? 

Let's also make a test that renders two different fixtures, and one that attempts to specify a filename that is not inside the deploy dir.

Reply:
> This is probably rendering the same thing twice, right? Should we be de-duping the file names?

Yeah. In line with `cat` behaviour, I think we should actually render the same file twice. Things taking the output of `kubernetes-render` are likely going to expect a 1-1 mapping between input filenames and rendered content, and I think silently dropping arguments (even if they resolve to be duplicates) could be pretty unexpected and cause downstream complexity. I would vote for keeping it simple and doing what the invoker said even if it seems weird.

Added those tests, thanks.